File: Module.Frontend.TPM/Client/model/tpm/mechanictype/HistoricalMechanicType.js
Ext.define('App.model.tpm.mechanictype.HistoricalMechanicType', {
    extend: 'Ext.data.Model',
    idProperty: '_Id',
    breezeEntityType: 'HistoricalMechanicType',
    fields: [
        { name: '_Id', type: 'string', hidden: true },
        { name: '_ObjectId', hidden: true },
        { name: '_User', type: 'string', isDefault: true },
        { name: '_Role', type: 'string', isDefault: true },
        { name: '_EditDate', type: 'date', isDefault: true, timeZone: +3, convert: dateConvertTimeZone},
        { name: '_Operation', type: 'string', isDefault: true },
        { name: 'Name', type: 'string', hidden: false, isDefault: true },
        { name: 'Discount', useNull: true, type: 'float', hidden: false, isDefault: true },
        { name: 'ClientTreeFullPathName', type: 'string', hidden: false, isDefault: true },
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'HistoricalMechanicTypes',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        }
    }
});

File: Module.Persist.TPM/Model/History/HistoricalMechanicType.cs
using Core.History;
using Module.Persist.TPM.Model.TPM;
using System;
namespace Module.Persist.TPM.Model.History
{
    [AssociatedWith(typeof(MechanicType))]
    public class HistoricalMechanicType : BaseHistoricalEntity<System.Guid>
    {
        public string Name { get; set; }
        public double? Discount { get; set; }
        public string ClientTreeFullPathName { get; set; }
    }
}
